test/language/expressions/tagged-template/invalid-escape-sequences.js

```diff
@@ -0,0 +1,70 @@
+// Copyright (C) 2016 Tim Disney. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: Invalid unicode escape sequence in tagged template literals are allowed
+es6id: 11.8.6
```

Comment:
It's a fairly minor change to an existing ES6 feature. I don't think any new sections were even added. I may be wrong, though.